XCUITests/AuthenticationTest.swift

```diff
@@ -263,4 +265,24 @@ class AuthenticationTest: BaseTestCase {
         waitForExistence(app.navigationBars["Enter Passcode"])
         navigator.performAction(Action.UnlockLoginsSettings)
     }
+
+    func testBasicHTTPAuthenticationPromptVisible() {
+        navigator.openURL(testBasicHTTPAuthURL)
+
+        waitForExistence(app.staticTexts["Authentication required"])
+        waitForExistence(app.staticTexts["A username and password are being requested by jigsaw.w3.org. The site says: test"])
+
+        let placeholderValueUsername = app.alerts.textFields.element(boundBy: 0).value as! String
+        let placeholderValuePassword = app.alerts.secureTextFields.element(boundBy: 0).value as! String
+
+        XCTAssertEqual(placeholderValueUsername, "Username")
+        XCTAssertEqual(placeholderValuePassword, "Password")
+
+        waitForExistence(app.alerts.buttons["Cancel"])
+        waitForExistence(app.alerts.buttons["Log in"])
+
+        // Dismiss authentication prompt
```

Comment:
I would also add a comment that we are dismissing and not trying to login too due to the crash issue. So that in the future, when that's fixed, we can complete the test

Reply:
I didn't even know about a crash issue. Is there a reference? (For what it's worth no credentials are supplied in the demo https://jigsaw.w3.org/HTTP/ – but at that point beyond the UI it becomes a WebKit/Unit test, as we only care about the UI which is the prompt.

Reply:
yeah, that's right. Thanks!